application/src/main/java/org/opentripplanner/framework/application/OTPFeature.java

```diff
@@ -38,6 +38,14 @@ public enum OTPFeature {
     "Should there be a transfer leg when transferring on the very same stop. Note that for in-seat/interlined transfers no transfer leg will be generated."
   ),
   FloatingBike(true, false, "Enable floating bike routing."),
+  GtfsCoach(
+    false,
+    false,
+    """
+    When parsing GTFS data, treat GTFS route type 200 to 299 as coach routes instead of bus routes.
+    When using the GTFS GraphQL API, do not return coach routes when only BUS is specified as a mode.
```

Comment:
I guess since you are only using this for reading in input data, this would also affect the behaviour of the transmodel API if you are using GTFS data?

Reply:
Yes. If you are reading 200-299, coach will be returned in both API if the feature is on.

Reply:
In that case, we probably should not specify that specific API in this text but talk about APIs in general.